browser-extension/styles/inject.css

```
@@ -0,0 +1,244 @@
+    animation-delay: 0s !important;
+}
+.mediafusion-upload-btn .mediafusion-dot:nth-child(2) {
+    animation-delay: 0.2s !important;
+}
+.mediafusion-upload-btn .mediafusion-dot:nth-child(3) {
+    animation-delay: 0.4s !important;
+}
+
+@keyframes mediafusion-wave {
+    0%, 60%, 100% {
+        transform: scale(0.8) !important;
```

Biome (2.1.2)

[error] Using !important within keyframes declaration is completely ignored in some browsers.

Consider removing useless !important declaration.

(lint/suspicious/noImportantInKeyframe)